db/fixtures/miq_policy_sets.yml

```diff
@@ -105,7 +105,24 @@
           description: Initiate SmartState Analysis for Container Image
           action_type: default
           options: {}
-      Condition: []
+      Condition:
+        - name: do not scan image-inspectors image
+          description: Dont scan image-inspectors image
+          modifier: deny
+          expression: !ruby/object:MiqExpression
+            exp:
+              ENDS WITH:
+                field: ContainerImage-name
+                value: "image-inspector"
+            context_type:
+          towhat: ContainerImage
+          file_mtime:
+          guid: e744245a-3d08-11e6-8d39-02422087d789
```

Comment:
Thinking more, the problem is not storing info on the image / event.
Can look it up instead when the policy runs.  Not sure how exactly to expose a method to conditions, but I suspect virtual_column and/or a scope might work...

However this again has an order-dependence problem.  If the condition is only evaluated when the Image is created, and if it's first created due to us running a scan, it won't later get scanned when a user runs same image.

=> The problem is having an event that fires again when a non-our-scanning pod is added.
Probably "Pod created" (oh, wait, I need to finish that first) — but then need other mechanism to avoid scanning many times (while still allowing manually triggerred re-scans)...

- Maybe we don't care about the order dependency?  
  The reason we want to tighten up the condition in the first place is that users might have other images named 'image-inspector', right?  If they're distinct images, that's still easy; only gets tricky if they independently run the *very same* image.